saleor/graphql/account/mutations/permission_group.py

```diff
@@ -85,23 +89,48 @@ def clean_permissions(
         cls,
         requestor: "User",
         errors: Dict[Optional[str], List[ValidationError]],
-        field: str,
         cleaned_input: dict,
     ):
+        field = "add_permissions"
         permission_items = cleaned_input.get(field)
         if permission_items:
-            cleaned_input[field] = get_permissions(cleaned_input[field])
+            cleaned_input[field] = get_permissions(permission_items)
+            cls.can_manage_permissions(requestor, errors, field, permission_items)
 
-            if requestor.is_superuser:
-                return
+    @classmethod
+    def can_manage_permissions(
```

Comment:
What about `ensure_can_manage_permissions`? I expect `can_*` to return a bool

Reply:
Done :)